### PR TITLE
Documentation shop plan Softlayer cloud server

### PR DESCRIPTION
**Pull request by RamonHossein**
- Identification
- User
  - User Organization Profile (Show and Update)
  - User Organization Profile Telephone (Create, Update, Show and List) 
  - User Organization Profile Address (Create, Update and Show)
  - User Select Products and Add Them To His Shopping Cart
- Admin
  - List Shop Plan Softlayer Cloud Server
  - Update Shop Plan Softlayer Cloud Server
  - Show Shop Plan Softlayer Cloud Server

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -3025,6 +3025,215 @@ processor | String | 4 x 2.0 GHz Cores
...
+    }
+  ]
+}
+```
+
+```ruby
+Not implemented
+```
+
+The admin can visualize all plans Softlayer cloud server created by accessing the route below.
```
Comment (fernandes) [inline source/index.md]: The admin can list all SoftLayer Cloud Server plans accessing the route

File: source/index.md [hunk 2]
```diff
@@ -3025,6 +3025,215 @@ processor | String | 4 x 2.0 GHz Cores
...
+    }
+  }
+}
+```
+
+```ruby
+Not implemented
+```
+
+The admin can visualize a specific plan Softlayer cloud server created by accessing the route below.
```
Comment (fernandes) [inline source/index.md]: The admin can get information for a specific SoftLayer Cloud Server plan accessing the route

File: source/index.md [hunk 3]
```diff
@@ -3025,6 +3025,215 @@ processor | String | 4 x 2.0 GHz Cores
...
+    }
+  }
+} 
+```
+
+```ruby
+Not implemented
+```
+
+The admin can edit a plan Softlayer cloud server by accessing the route below.
```
Comment (fernandes) [inline source/index.md]: The admin can edit a SoftLayer Cloud Server plan accessing the route

**Comment by RamonHossein** [comment]
@fernandes I add the documentation user organization profile documentation

File: source/index.md [hunk 4]
```diff
@@ -61,34 +61,11 @@ horus-cli version
...
-id-admin-catalog-provider-product-option | 37098c00-7215-5752-8c65-738d04ea4a02
-id-manager-profile | 1c9dcbb4-758d-55c0-874a-da7a8285a814
-id-manager-company | 78473988-b021-4864-aa48-f3b7bc4a06f6
-id-manager-company-telephone | 7915969d-e949-4066-b264-8d5bdcdf2d5c
-id-manager-company-address | 1906c99a-8d54-42e1-a035-cc425553f1fd
-id-manager-client-user-profile | 1b5c1105-360c-418c-8362-385749ec5746
-id-manager-client-user | 03b829d7-bbe6-5556-ae91-e480c27a28ee
-id-manager-client-user-telephone | 690c7235-59b3-5a4e-9c04-e1aa98cc6e38
-id-manager-client-user-address | 0778b605-36ca-4a4e-8f54-d38f830d226c
+All system identification is a UUID type and it begins with a substring "id + resource + function".
```
Comment (fernandes) [inline source/index.md:64]: good! :+1:

**Pull request merged**